vllm/engine/arg_utils.py

```diff
@@ -1105,7 +1121,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             "--ec-transfer-config", **vllm_kwargs["ec_transfer_config"]
         )
         vllm_group.add_argument(
-            "--compilation-config", "-O", **vllm_kwargs["compilation_config"]
+            "--compilation-config",
+            "-cc",
+            "-O",
```

Comment:
Yeah let's remove the `-O` here completely and just remap any use of `-O.<...>`/`-O {...}` to `-cc` in `argparse_utils`